File: group.c
#include "group.h"

int create_group(char group_name[80])
{
    group new_group;
    strcpy(new_group.group_name, group_name);
    printf("%s", group_name);
    return NO_ERROR;
}

int delete_group(char group_name[GROUPNAME_LEN])
{
    group new_group;
    /* TODO: Add delete functionality */
    return NO_ERROR;
}

group* view_groups()
{
    /* TODO: Add group lookup functionality */
    return NO_ERROR;
}

int get_group_id(group g)
{
    
}

int main()
{
    char gr[80] = "My New Group";
    create_group(gr);
    return 0;
}

File: group.py
from constants import error as err

class Group:
    def __init__(self, group_name = '',
                group_id = -1, date_created = ''):
        return 

    def create_group(self, group_name,
                    group_id, date_created):
        self.group_name = group_name
        self.group_id = group_id
        self.date_created = date_created
        return err["NO_ERROR"]

    def get_group_id(self, id):
        self.group_id = id
        return err["NO_ERROR"]

    def view_groups(self):
        # TODO: Add method to view groups
        pass
